spec/restaurant_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'item'
+require 'customer'
+
+describe 'restaurant' do
+
+  subject {Restaurant.new}
+
+  it 'creates a restaurant' do
+    expect(subject).to be_kind_of Restaurant
+  end
+
+    it 'creates a menu' do
+    expect(subject.menu).to be_kind_of Hash
+  end
```

Comment:
Lint/BlockAlignment: `end` at 14, 2 is not aligned with `it 'creates a menu' do` at 12, 4